docs/proposals/2018-10-29 Restricted_Content.md

```diff
@@ -0,0 +1,35 @@
+# Problem
+
+Some content can be restricted due to various reasons. We want to leave a placeholder for when these omissions occur with a message as to the reason why.
+
+# Proposal
+
+We propose that a new restricted content element can replace the content element when it is removed.
+
+## Example
+```json
+{
+  "_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  "type": "story",
```

Comment:
Is this type wrong? Looking at #177 should the type here be `restricted_content`?  Otherwise I'm confused why `reasons` is not nested inside of `restricted_content`.

Also, can we get some clarification/documentation included (maybe in #177 implementation) on how this is supposed to work?  Can a request for any content object return a `restricted_content` object instead of the actual object? Can this happen only at the top level, or can nested content elements be restricted and return one of these?  Is this `_id` the ID of the requested content object or a separate ID for the restriction?  This seems like a case that really needs to be documented clearly.

Reply:
any request for a content object can have a restricted_content object returned instead. this can happen at the top level or any level of nesting, and will elide all other content elements within it. 

the top level id is the restricted content id. the id for the restriction is the id in the reasons field.